src/moj/components/date-picker/date-picker.js

```diff
@@ -0,0 +1,948 @@
+    const disabled = this.isExcludedDate(thisDay);
+
+    this.calendarDays[i].update(thisDay, hidden, disabled);
+
+    thisDay.setDate(thisDay.getDate() + 1);
+  }
+};
+
+Datepicker.prototype.setCurrentDate = function (focus = true) {
+  const { currentDate } = this;
+
+  this.calendarDays.forEach((calendarDay) => {
+    calendarDay.button.classList.add("moj-datepicker-button");
+    calendarDay.button.classList.add("moj-datepicker-calendar__day");
+    calendarDay.button.setAttribute("tabindex", -1);
+    calendarDay.button.classList.remove("selected");
+    const calendarDayDate = calendarDay.date;
+    calendarDayDate.setHours(0, 0, 0, 0);
+
+    const today = new Date();
+    today.setHours(0, 0, 0, 0);
+
+    if (
+      calendarDayDate.getTime() ===
+      currentDate.getTime() /* && !calendarDay.button.disabled */
+    ) {
+      if (focus) {
+        calendarDay.button.setAttribute("tabindex", 0);
+        calendarDay.button.focus();
+        calendarDay.button.classList.add("selected");
+      }
+    }
+
+    if (
+      this.inputDate &&
+      calendarDayDate.getTime() === this.inputDate.getTime()
+    ) {
+      calendarDay.button.classList.add("current");
+      calendarDay.button.setAttribute("aria-selected", true);
+    } else {
+      calendarDay.button.classList.remove("current");
+      calendarDay.button.removeAttribute("aria-selected");
+    }
+
+    if (calendarDayDate.getTime() === today.getTime()) {
+      calendarDay.button.classList.add("today");
+    } else {
+      calendarDay.button.classList.remove("today");
+    }
+  });
+
+  // if no date is tab-able, make the first non-disabled date tab-able
+  if (!focus) {
+    const enabledDays = this.calendarDays.filter((calendarDay) => {
+      return (
+        window.getComputedStyle(calendarDay.button).display === "block" &&
+        !calendarDay.button.disabled
+      );
+    });
+
+    enabledDays[0].button.setAttribute("tabindex", 0);
+
+    this.currentDate = enabledDays[0].date;
+  }
+};
+
+Datepicker.prototype.selectDate = function (date) {
+  if (this.isExcludedDate(date)) {
+    return;
+  }
+
+  this.$calendarButton.querySelector("span").innerText =
+    `Choose date. Selected date is ${this.formattedDateHuman(date)}`;
+  this.$input.value = this.formattedDateFromDate(date);
+
+  const changeEvent = new Event("change", { bubbles: true, cancelable: true });
+  this.$input.dispatchEvent(changeEvent);
+
+  this.closeDialog();
+};
+
+Datepicker.prototype.isOpen = function () {
+  return this.$dialog.classList.contains("moj-datepicker-dialog--open");
+};
+
+Datepicker.prototype.toggleDialog = function (event) {
+  event.preventDefault();
+  if (this.isOpen()) {
+    this.closeDialog();
+  } else {
+    this.setMinAndMaxDatesOnCalendar();
+    this.openDialog();
+  }
+};
+
+Datepicker.prototype.openDialog = function () {
+  // display the dialog
+  this.$dialog.style.display = "block";
+  this.$dialog.classList.add("moj-datepicker-dialog--open");
```

Comment:
`.moj-datepicker-dialog--open` doesn't exist in the CSS, but I imagine it should be
```css
.moj-datepicker-dialog--open {
  display: block;
}
```
So you could get rid of the manual `style` override